feedback.md

```diff
@@ -0,0 +1,23 @@
+# API Muncher
+## What We're Looking For
+
+| Feature | Feedback    |
+| :------------- | :------------- |
+| **Core Requirements** |   |
+| Git hygiene |  |
+| Comprehension questions	|  |
+| **General** |  |
+| Rails fundamentals (RESTful routing, use of named paths) |  |
+| Semantic HTML |  |
+| Errors are reported to the user |  |
+| API Wrapper to handle the API requests |  |
+| Controller testing |  |
+| **Styling** |  |
+| Foundation Styling for responsive layout |  |
+| Search View |  |
+| List View |  |
+| Short View |  |
+| **Styling** |  |
+| Foundation is used appropriately |  |
+| Look and feel is similar to the original |  |
+|  **Overall** |  |
```

Comment:
We should add something for API attribution (since that's in the requirements for using the API)